notes.md

```diff
@@ -0,0 +1,23 @@
+# SQL Injection
+
+Simples sql injection is possible by putting `'OR 1=1) -- '` in the search bar
+
+### Thought process
+
+Could be good to allow user to look into the query syntax somehow on error (return sql errors straight to the fronted?)
```

Comment:
można spróbwać to https://stackify.com/display-php-errors/ 
albo wrzucić db->query() w blok try catch i zwrócić error message w catchu, tylko też trzeba obsłużyć na froncie wtedy